index.js

```diff
@@ -38,7 +42,32 @@
         Update `gradeTotal` so it increases value for each item in the array
 
  */
- const gradeTotal = 0
+ // expect this to be the sum of all my `total` values
+let gradeTotal = 0
+
+// option 1
+for (const index in grades) {
+    const { score } = grades[index]
+    console.log({ score })
```

Comment:
Can we make this into a single line since this code is only for visibility of what the program is doing? I am thinking something like

`console.log(score = grades[index])`